components/alert/alert-toast.js

```diff
@@ -71,6 +70,15 @@ class AlertToast extends LitElement {
 			d2l-alert {
 				animation: none;
 			}
+
+			@media (max-width: 615px) {
```

Comment:
This doesn't need to be in a media query since the max width of a toast is `600px`

Reply:
@Scy-D2L and I talked a bit and she correctly pointed out that the `bottom: 12px` is still necessary in the media query.